src/pages/projectSettings/CopyProjectModal.tsx

```diff
@@ -47,11 +47,6 @@ export const CopyProjectModal: React.VFC<Props> = ({
         `There was an error creating the project: ${err.message}`
       );
     },
-    refetchQueries: [
-      "GetViewableProjectRefs",
-      "ProjectSettings",
-      "RepoSettings",
-    ],
```

Comment:
We no longer need to refetch these queries because the modal now redirects to the new project on success. This results in a new call being made to the backend.